### ISSUE TITLE
core::133::http-uri-parent false positive for long-running creates of top-level resources

### ISSUE THREAD

**Issue by rofrankel** [googleapis/api-linter#1022]
When creating a top-level resource is a long-running operation, this check in rules/aip-133/http_uri_parent.go incorrectly runs the rule:

```
	OnlyIf: func(m *desc.MethodDescriptor) bool {
		return isCreateMethod(m) && !hasNoParent(m.GetOutputType())
	}
```

One potential fix would be to update `hasNoParent` to return true specifically for `Operation`, but this feels like a hack (on top of a hack, arguably), since `m.GetOutputType()` isn't actually the correct way to determine the resource being created.

However, I don't know what the right way _is_ to get the type of the resource.  Perhaps `GetOutputType` should know how to parse `operation_info` and look at `response_type`?  But I'm not sure where else `GetOutputType` might be used with the assumption that it does indeed return `Operation` for LROs.  Suggestions from someone more familiar with the codebase would be appreciated!

**Comment by noahdietz** [comment]
Can you give me an example proto that would reproduce the issue?

**Comment by rofrankel** [comment]
Thank you!
